Source/AtomicEditor/Editors/SceneEditor3D/SceneEditor3D.h

```diff
@@ -22,6 +22,9 @@
 
 #pragma once
 
+#include <Atomic/UI/SystemUI/SystemUI.h>
+#include <Atomic/UI/SystemUI/DebugHud.h>
+
```

Comment:
Including SystemUI and DebugHud in to the SceneEditor header, can these be brought into the .cpp file instead, always opt for including in the source file as this helps keep the build fast and helps prevent breakage... additional includes also erode encapsulation and are often a sign the API can be better